src/api/db/migrate/20170111114943_add_exclude_projects_to_configuration.rb

```diff
@@ -0,0 +1,17 @@
+class AddExcludeProjectsToConfiguration < ActiveRecord::Migration[5.0]
+  def self.up
+    add_column :configurations, :excluded_projects, :string
+    add_column :configurations, :excluded_projects_description, :string
+
+    # update database from file
+    old = CONFIG['global_write_through']
+    CONFIG['global_write_through'] = false
+    ::Configuration.first.update_from_options_yml
+    CONFIG['global_write_through'] = old
```

Comment:
This line is a bit confusing to me, I don't see whats the point of setting this constant here?